integration_tests/pages/page.ts

```diff
@@ -1,19 +1,53 @@
+import 'cypress-axe'
+import { Result } from 'axe-core'
+
 export type PageElement = Cypress.Chainable<JQuery>
 
 export default abstract class Page {
   static verifyOnPage<T>(constructor: new () => T): T {
     return new constructor()
   }
 
-  constructor(private readonly title: string) {
-    this.checkOnPage()
+  constructor(
+    private readonly title: string,
+    private readonly skipA11y = false,
+  ) {
+    if (title) {
```

Comment:
why the check here?  Isn't `title` always going to be set?

Reply:
Copied from other project without thinking about it much - removed!